### PR TITLE
DOC: add english doc 'Pod scheduling optimization'

### PR DESCRIPTION
**Pull request by zhang-x-z**
Signed-off-by: ZhangXiaozheng <zhang__xiaozheng@outlook.com>

### Ⅰ. Describe what this PR does
Add english document 'Pod scheduling optimization'.

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

File: docs/en/TOC.md [hunk 1]
```diff
@@ -31,6 +31,7 @@
 + Advanced   
   - [Accelerate Data Access by MEM or SSD](samples/accelerate_data_by_mem_or_ssd.md)
   - [Alluxio Tieredstore Configuration](samples/tieredstore_config.md)
+  - [Optimize Pod scheduling through Webhook mechanism](operation/pod_schedule_global.md)
```
Comment (cheyang) [inline docs/en/TOC.md:34]: Please rename pod_schedule_global.md to pod_schedule_optimization.md

File: docs/en/operation/pod_schedule_global.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
+# Demo - Pod Scheduling Optimization
```
[the remark this answers is not in the document]
Comment (zhang-x-z) [reply docs/en/operation/pod_schedule_global.md:1]: Should I change the title to just 'Optimization', or keep the title same as the file name (Pod Schedule Optimization)? Please be more detailed, thanks.

File: docs/en/TOC.md (hunk 1 above)
Comment (TrafalgarZZZ) [inline docs/en/TOC.md:34]: Suggest renaming it to `Optimize Pod Scheduling` or `Pod Scheduling Optimization`

File: docs/en/operation/pod_schedule_global.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      - preference:
+          matchExpressions:
+          - key: fluid.io/s-default-hbase
+            operator: In
+            values:
+            - "true"
+        weight: 100
+```
+
+Through the Webhook mechanism, the application Pod is injected and caches the weak affinity configuration of the worker.
```
Comment (TrafalgarZZZ) [inline docs/en/operation/pod_schedule_global.md:161]:
```suggestion
Through the Webhook mechanism, the application Pod is injected with preferred affinity to the cache worker.
```
"preferred" is recommended here.

**Pull request merged**